### DISCUSSION TITLE
Pimcore 11 - File Explorer bundle

### DISCUSSION THREAD

**Discussion by ravi-credencys** [pimcore/pimcore#15544] [Q&A / Support] [answered]
I have installed Pimcore 11 File Explorer bundle from pimcore/file-explorer-bundle using below command.

`composer require pimcore/file-explorer-bundle`

I have added class in config/bundles.php

```
return [
        Pimcore\Bundle\FileExplorerBundle\PimcoreFileExplorerBundle::class => ['all' => true]
];
```

but still file explorer is not visible in Pimcore 11 Admin.
Anyone have any input on this?

Thanks

**Comment by blankse** [comment, marked as answer]
Did you also install the bundle?
`php bin/console pimcore:bundle:install PimcoreFileExplorerBundle`
The user then needs the `fileexplorer` permission.

**Reply by ravi-credencys** [reply]
Hello @blankse,

Thanks for the reply,
Yes, I have executed that bundle install command but as you mentioned, there were some permission issue was there.
Now it's resolved.
Thanks once again.
